src/pydata_sphinx_theme/theme/pydata_sphinx_theme/_templates/version-switcher.html

```diff
@@ -72,6 +72,12 @@
             if (entry.version == "{{ theme_switcher.get('version_match') }}") {
                 node.classList.add("active");
                 $("#version_switcher_button").text(entry.name);
+                // Add extra class to the button for the selected entry
+                if ("extra_classes" in entry) {
+                    $.each(entry.extra_classes, function(index, extra_class) {
```

Comment:
```suggestion
                    $.each(entry.extra_classes, function(index, extra_class) {
                        if (typeof entry.extra_classes === "string") {
                            entry.extra_classes = [entry.extra_classes]
                        }
```

Reply:
Should we do something like this in case somebody just gives a string?

Reply:
Personally I would not add complexity here. But I can make the change if you want.

I think the suggestion is not correct as the type check should happen before iterating on the list of classes. (But I am really a novice on JS so I could be missing something as well)

Reply:
OK sounds good, let me re-work this so it's a one-liner then